{"uid":"e7ff243ddece9f45","name":"Login with valid credentials test","fullName":"tests.login_test.TestLogin#test_valid_login","historyId":"18657c207d70aa57845e0b7f4751c290","time":{"start":1702948511036,"stop":1702948521296,"duration":10260},"description":"valid login","descriptionHtml":"<p>valid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564a3e061d33 <unknown>\n#1 0x564a3dd1ef87 <unknown>\n#2 0x564a3dd6ec0e <unknown>\n#3 0x564a3dd6ed11 <unknown>\n#4 0x564a3ddba004 <unknown>\n#5 0x564a3dd9836d <unknown>\n#6 0x564a3ddb7297 <unknown>\n#7 0x564a3dd980e3 <unknown>\n#8 0x564a3dd60044 <unknown>\n#9 0x564a3dd6144e <unknown>\n#10 0x564a3e026861 <unknown>\n#11 0x564a3e02a785 <unknown>\n#12 0x564a3e014285 <unknown>\n#13 0x564a3e02b41f <unknown>\n#14 0x564a3dff820f <unknown>\n#15 0x564a3e04f028 <unknown>\n#16 0x564a3e04f1f7 <unknown>\n#17 0x564a3e060ed4 <unknown>\n#18 0x7f2824a94ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f2ca3b48d90>\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.flaky(reruns=1)\n    def test_valid_login(self, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:42: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"e2f19dacbc8fdcf4c3c3fcbf7245b77c\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f2ca3be8d60>\nmessage = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x564a3e061d33 <unknown>\nE       #1 0x564a3dd1ef87 <unknown>\nE       #2 0x564a3dd6ec0e <unknown>\nE       #3 0x564a3dd6ed11 <unknown>\nE       #4 0x564a3ddba004 <unknown>\nE       #5 0x564a3dd9836d <unknown>\nE       #6 0x564a3ddb7297 <unknown>\nE       #7 0x564a3dd980e3 <unknown>\nE       #8 0x564a3dd60044 <unknown>\nE       #9 0x564a3dd6144e <unknown>\nE       #10 0x564a3e026861 <unknown>\nE       #11 0x564a3e02a785 <unknown>\nE       #12 0x564a3e014285 <unknown>\nE       #13 0x564a3e02b41f <unknown>\nE       #14 0x564a3dff820f <unknown>\nE       #15 0x564a3e04f028 <unknown>\nE       #16 0x564a3e04f1f7 <unknown>\nE       #17 0x564a3e060ed4 <unknown>\nE       #18 0x7f2824a94ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[{"name":"json_data","time":{"start":1702948326939,"stop":1702948326939,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1702948326939,"stop":1702948326939,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1702948326939,"stop":1702948326939,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"valid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564a3e061d33 <unknown>\n#1 0x564a3dd1ef87 <unknown>\n#2 0x564a3dd6ec0e <unknown>\n#3 0x564a3dd6ed11 <unknown>\n#4 0x564a3ddba004 <unknown>\n#5 0x564a3dd9836d <unknown>\n#6 0x564a3ddb7297 <unknown>\n#7 0x564a3dd980e3 <unknown>\n#8 0x564a3dd60044 <unknown>\n#9 0x564a3dd6144e <unknown>\n#10 0x564a3e026861 <unknown>\n#11 0x564a3e02a785 <unknown>\n#12 0x564a3e014285 <unknown>\n#13 0x564a3e02b41f <unknown>\n#14 0x564a3dff820f <unknown>\n#15 0x564a3e04f028 <unknown>\n#16 0x564a3e04f1f7 <unknown>\n#17 0x564a3e060ed4 <unknown>\n#18 0x7f2824a94ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f2ca3b48d90>\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.flaky(reruns=1)\n    def test_valid_login(self, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:42: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"e2f19dacbc8fdcf4c3c3fcbf7245b77c\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f2ca3be8d60>\nmessage = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x564a3e061d33 <unknown>\nE       #1 0x564a3dd1ef87 <unknown>\nE       #2 0x564a3dd6ec0e <unknown>\nE       #3 0x564a3dd6ed11 <unknown>\nE       #4 0x564a3ddba004 <unknown>\nE       #5 0x564a3dd9836d <unknown>\nE       #6 0x564a3ddb7297 <unknown>\nE       #7 0x564a3dd980e3 <unknown>\nE       #8 0x564a3dd60044 <unknown>\nE       #9 0x564a3dd6144e <unknown>\nE       #10 0x564a3e026861 <unknown>\nE       #11 0x564a3e02a785 <unknown>\nE       #12 0x564a3e014285 <unknown>\nE       #13 0x564a3e02b41f <unknown>\nE       #14 0x564a3dff820f <unknown>\nE       #15 0x564a3e04f028 <unknown>\nE       #16 0x564a3e04f1f7 <unknown>\nE       #17 0x564a3e060ed4 <unknown>\nE       #18 0x7f2824a94ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1702948511037,"stop":1702948521296,"duration":10259},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564a3e061d33 <unknown>\n#1 0x564a3dd1ef87 <unknown>\n#2 0x564a3dd6ec0e <unknown>\n#3 0x564a3dd6ed11 <unknown>\n#4 0x564a3ddba004 <unknown>\n#5 0x564a3dd9836d <unknown>\n#6 0x564a3ddb7297 <unknown>\n#7 0x564a3dd980e3 <unknown>\n#8 0x564a3dd60044 <unknown>\n#9 0x564a3dd6144e <unknown>\n#10 0x564a3e026861 <unknown>\n#11 0x564a3e02a785 <unknown>\n#12 0x564a3e014285 <unknown>\n#13 0x564a3e02b41f <unknown>\n#14 0x564a3dff820f <unknown>\n#15 0x564a3e04f028 <unknown>\n#16 0x564a3e04f1f7 <unknown>\n#17 0x564a3e060ed4 <unknown>\n#18 0x7f2824a94ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 38, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 101, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"3bc554887bfcdccb","name":"public ip address","source":"3bc554887bfcdccb.txt","type":"text/plain","size":13},{"uid":"ba675929b207f3","name":"Cookies","source":"ba675929b207f3.json","type":"application/json","size":3357},{"uid":"bf06103838e850df","name":"Session Storage","source":"bf06103838e850df.json","type":"application/json","size":358},{"uid":"cc1b660bec93206d","name":"Local Storage","source":"cc1b660bec93206d.json","type":"application/json","size":208},{"uid":"638bd9d86a8ababc","name":"Console Logs","source":"638bd9d86a8ababc.json","type":"application/json","size":2},{"uid":"224f43b965943250","name":"Full Page Screenshot","source":"224f43b965943250.png","type":"image/png","size":1275914},{"uid":"f322ac824c6695f1","name":"URL","source":"f322ac824c6695f1.uri","type":"text/uri-list","size":23},{"uid":"d84a4cfa448e03e2","name":"Network Logs","source":"d84a4cfa448e03e2.json","type":"application/json","size":25827}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"blocker"},{"name":"epic","value":"Security"},{"name":"tag","value":"Tagged test"},{"name":"feature","value":"Login"},{"name":"tag","value":"flaky(reruns=1)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az1118-425"},{"name":"thread","value":"2283-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[{"uid":"8301f2e054ef602f","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x557c1cafdd33 <unknown>\n#1 0x557c1c7baf87 <unknown>\n#2 0x557c1c80ac0e <unknown>\n#3 0x557c1c80ad11 <unknown>\n#4 0x557c1c856004 <unknown>\n#5 0x557c1c83436d <unknown>\n#6 0x557c1c853297 <unknown>\n#7 0x557c1c8340e3 <unknown>\n#8 0x557c1c7fc044 <unknown>\n#9 0x557c1c7fd44e <unknown>\n#10 0x557c1cac2861 <unknown>\n#11 0x557c1cac6785 <unknown>\n#12 0x557c1cab0285 <unknown>\n#13 0x557c1cac741f <unknown>\n#14 0x557c1ca9420f <unknown>\n#15 0x557c1caeb028 <unknown>\n#16 0x557c1caeb1f7 <unknown>\n#17 0x557c1cafced4 <unknown>\n#18 0x7f4c10894ac3 <unknown>","time":{"start":1702948497835,"stop":1702948508082,"duration":10247}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["security","Tagged test","flaky(reruns=1)"]},"source":"e7ff243ddece9f45.json","parameterValues":[]}